content-src/asrouter/templates/Trailhead/Trailhead.jsx

```diff
@@ -233,30 +226,29 @@ export class _Trailhead extends React.PureComponent {
       "trailhead",
       content && content.className,
     ].filter(v => v).join(" ");
+
     return (<>
     {this.state.isModalOpen && content ? <ModalOverlayWrapper innerClassName={innerClassName} onClose={this.closeModal} id="trailheadDialog" headerId="trailheadHeader">
       <div className="trailheadInner">
         <div className="trailheadContent">
           <h1 data-l10n-id={content.title.string_id}
-            id="trailheadHeader">{this.getStringValue(content.title)}</h1>
+            id="trailheadHeader" />
```

Comment:
@Mardak instead of using FluentOrText can we just change property_id => string_id in OnboardingMessageProvider.jsm and use normal fluent?

Reply:
Ah yup plain fluent should be good as we aren't passing in raw strings here.